### PR TITLE
Added Note about arm7 version to doc

### PR DESCRIPTION
**Pull request by suisseWalter**
This pull request adds a troubleshooting subsection for the non working arm7 release on raspberry Pi's in the installation part of the documentation. 
This pull request was suggested in issue #6979

### CODE REVIEW

**Comment by mrsdizzie** [comment]
This feels wrong -- I still think we should just remove the arm7 version from the downloads section if it is known to not work for a while (and reported regularly). It doesn't make sense to offer something known to not work with a note that says "this does not work" buried in one of the docs.

**Comment by suisseWalter** [comment]
yes, I do agree with that but I don't have any idea on how to change that. therefore I did this. so if someone just removes arm7 I would be really happy. also is arm7 completely broken or only on raspbian?

**Comment by lunny** [comment]
I like @mrsdizzie 's idea. We can manually delete exist arm-7 binaries and change the Makefile to avoid to compile arm7 binary.

**Comment by atomi** [comment]
Why is it even broken in the first place?

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
imho it is good to have info in docs while we have not fixed the issue

File: docs/content/doc/installation/from-binary.en-us.md [hunk 1]
```diff
@@ -142,7 +142,15 @@ For errors like `702 runWeb()] [E] Failed to start server: listen tcp 0.0.0.0:30
 bind: address already in use` Gitea needs to be started on another free port. This
 is possible using `./gitea web -p $PORT`. It's possible another instance of Gitea
 is already running.
+### Gitea not running on Raspbian
```
Comment (zeripath) [inline docs/content/doc/installation/from-binary.en-us.md]:
```suggestion

### Running Gitea on Raspbian
```

File: docs/content/doc/installation/from-binary.en-us.md [hunk 2]
```diff
@@ -142,7 +142,15 @@ For errors like `702 runWeb()] [E] Failed to start server: listen tcp 0.0.0.0:30
...
 
+currently, as of v1.8, the arm7 version of gitea is broken and doesn't run on Raspberry Pi's and similar devices.
```
Comment (zeripath) [inline docs/content/doc/installation/from-binary.en-us.md]:
```suggestion
As of v1.8, there is a problem with the arm7 version of Gitea and it doesn't run on Raspberry Pi and similar devices. 
```

File: docs/content/doc/installation/from-binary.en-us.md [hunk 3]
```diff
@@ -142,7 +142,15 @@ For errors like `702 runWeb()] [E] Failed to start server: listen tcp 0.0.0.0:30
 bind: address already in use` Gitea needs to be started on another free port. This
 is possible using `./gitea web -p $PORT`. It's possible another instance of Gitea
 is already running.
+### Gitea not running on Raspbian
 
+currently, as of v1.8, the arm7 version of gitea is broken and doesn't run on Raspberry Pi's and similar devices. 
+
+Therefore switch to the arm6 version, which works on Raspberry Pi's.
```
Comment (zeripath) [inline docs/content/doc/installation/from-binary.en-us.md]:
```suggestion
It is therefore recommended to switch to the arm6 version which has been tested and shown to work on Raspberry Pi and similar devices.
```

**Comment by zeripath** [comment]
@suissewalter if you accept my suggested changes I'll approve. Even if we stop building the broken arm 7 builds we'll need to say something and this would do.

**Review by zeripath — APPROVED** [review]

**Comment by zeripath** [comment]
make LG-TM work

**Pull request merged**